core/spec/models/spree/product_spec.rb

```diff
@@ -477,6 +477,18 @@ class Extension < Spree::Base
         expect(product.possible_promotions).to include(promotion)
       end
     end
+
+    context "#discard" do
+      let(:product_property) { create(:product_property) }
+      let(:product) { product_property.product }
+
+      it "should not delete product properties objects" do
```

Comment:
Can you please avoid "should" here? `it "does not delete...` form is preferred.

Reply:
done https://github.com/solidusio/solidus/pull/3226/files#diff-72c8de7678be26819204ff95fe964da9R485